### ISSUE TITLE
Plans of Releasing the Training Code of ECGFounder?

### ISSUE THREAD

**Issue by King-HAW** [NickLJLee/ECGFounder#7]
Hi, thanks for releasing the model weights, the inference code, and the fine-tuning code. I want to ask if you guys have a plan to release the training code? Because the current description of the model training and the "positive label argument" in Fig.1 (Positive Unlabeled learning, especially the loss function) is not clear enough to reproduce the model training process. Much appreciate.
